exporter.go

```diff
@@ -967,14 +970,15 @@ func getKeysFromPatterns(c redis.Conn, keys []dbKeyPair) (expandedKeys []dbKeyPa
 	return expandedKeys, err
 }
 
-func (e *Exporter) connectToRedis(skipTLSVerification bool) (redis.Conn, error) {
+func (e *Exporter) connectToRedis(skipTLSVerification bool, tlsClientCertificates []tls.Certificate) (redis.Conn, error) {
```

Comment:
I was actually thinking, maybe we can just get rid of the two parameters to e.connectToRedis() and directly use `e.options.SkipTLSVerification` and `e.options.ClientCertificates` in the function.

Reply:
Good idea. Will do.

Reply:
Looks good now, thanks!